DataFormats/L1TrackTrigger/src/TTDTC.cc

```diff
@@ -40,7 +40,7 @@ void TTDTC::setStream(int dtcRegion, int dtcBoard, int dtcChannel, const StreamS
                 << "(" << dtcChannel << ") is out of range 0 to " << numOverlappingRegions_ - 1 << ".";
     throw exception;
   }
-  streams_[index(dtcRegion, dtcBoard, dtcChannel)] = move(stream);
+  streams_[index(dtcRegion, dtcBoard, dtcChannel)] = stream;
```

Comment:
that move got requested during code review as this code got merged into official release.

Reply:
Without this fix, the command "scram b code-checks-run" gives error:

/opt/ppd/cms/users/tomalin/bbb/CMSSW_12_0_0_pre4/src/DataFormats/L1TrackTrigger/src/TTDTC.cc:43:54: warning: std::move of the const variable 'stream' has no effect; remove std::move() or make the variable non-const [performance-move-const-arg]
  streams_[index(dtcRegion, dtcBoard, dtcChannel)] = move(stream);

The web site https://stackoverflow.com/questions/28595117/why-can-we-use-stdmove-on-a-const-object seems to confirm that move() has no effect if it acts on a const object. (Though for clang to say it is an error seems severe, since move() doesn't cause the code to crash).

Reply:
ah that one is also not the requested ones